nuxeo-services/nuxeo-platform-convert/src/main/java/org/nuxeo/ecm/platform/convert/plugins/UTF8CharsetConverter.java

```diff
@@ -70,7 +73,15 @@ protected Blob convert(Blob blob) throws IOException, ConversionException {
             return blob;
         }
         if (StringUtils.isEmpty(encoding)) {
-            try (InputStream in = blob.getStream()) {
+            Blob unencodedBlob = blob;
+            if (mimetype.equals(TEXT_HTML)) {
+                // convert to text to strip the html of any inline base64 resources
```

Comment:
The comment should explain *why* we do this, i.e., `to avoid mis-detecting the encoding`.